consul/datadog_checks/consul/consul.py

```diff
@@ -97,6 +97,7 @@ def __init__(self, name, init_config, instances):
         )
         self.disable_legacy_service_tag = is_affirmative(self.instance.get('disable_legacy_service_tag', False))
         self.service_whitelist = self.instance.get('service_whitelist', self.init_config.get('service_whitelist', []))
+        self.services_exclude = self.instance.get('services_exclude', self.init_config.get('services_exclude', []))
```

Comment:
nit: since this is used for lookup it would be better if it was a set

Reply:
Adding as set
```suggestion
        self.services_exclude = set(self.instance.get('services_exclude', self.init_config.get('services_exclude', [])))
```

Reply:
```suggestion
        self.services_exclude = set(self.instance.get('services_exclude', self.init_config.get('services_exclude', [])))
```